code/game/machinery/doors/_door.dm

```diff
@@ -446,17 +452,8 @@
 			source.thermal_conductivity = initial(source.thermal_conductivity)
 
 /obj/machinery/door/Move(new_loc, new_dir)
-	update_nearby_tiles()
-
 	. = ..()
-	if(width > 1)
-		if(dir in list(EAST, WEST))
-			bound_width = width * world.icon_size
-			bound_height = world.icon_size
-		else
-			bound_width = world.icon_size
-			bound_height = width * world.icon_size
-
+	update_nearby_tiles()
```

Comment:
Won't this fail to update all tiles at the original position if you do it post ..()?

Reply:
I think it should come before `..()` on further inspection, yes. The fact that it actually moves it is weird/bad, but whatever.